### DISCUSSION TITLE
Is this explicit override of library versions intended?

### DISCUSSION THREAD

**Discussion by gallardo** [serenity-bdd/serenity-core#2812] [General]
Reviewing the code I came through this section of a pom:

https://github.com/serenity-bdd/serenity-core/blob/1bec1b702dea8613211a084f6b7783efbb1829db/serenity-smoketests/pom.xml#L60-L75

that overrides the versions of the libraries defined globally for serenity-bdd. Is there a reason for this? It looks wrong, and in fact, the versions are not the same... Should the versioning not be delegated to the dependencyManagement of the parent pom?

**Comment by wakaleo** [comment]
The smoketests module is separate from the main project, so does not necessarily use the same dependencies.
